modules/db/src/main/scala/gem/dao/package.scala

```diff
@@ -64,8 +64,16 @@ package object dao {
       EitherConnectionIO.right[A, T](c)
   }
 
+  private def buildParam(param: (Any, (Put[_], NullabilityKnown))): Param.Elem = {
+    val (any, (put, nullability)) = param
+    nullability match {
+      case NoNulls => Param.Elem.Arg(any, put.asInstanceOf[Put[Any]])
+      case Nullable => Param.Elem.Opt(any.asInstanceOf[Option[_]], put.asInstanceOf[Put[Any]])
+    }
+  }
+
```

Comment:
Although this works, I imagine there's a better way to do this in `doobie` `0.8.x`. (@tpolecat ?)